app/precompiled.py

```diff
@@ -173,6 +164,25 @@ def sanitise_file_contents(encoded_string):
         }
 
 
+def rewrite_pdf(file_data, page_count):
+    file_data, recipient_address, redaction_failed_message = rewrite_address_block(file_data)
+
+    if not recipient_address:
+        raise ValidationFailed("address-is-empty", [1], page_count=page_count)
+
+    if not does_pdf_contain_cmyk(file_data) or does_pdf_contain_rgb(file_data):
+        file_data = convert_pdf_to_cmyk(file_data)
+
+    if contains_unembedded_fonts(file_data):
```

Comment:
Do we want to log when this happens? Do we care how many time we have to embed fonts?

Reply:
we log within the check function if it finds unembedded fonts

`current_app.logger.info(f'Found unembedded fonts {[x for x in unembedded]}')`